include/mbgl/gfx/uniform_block.hpp

```diff
@@ -101,31 +92,21 @@ class UniformBlockArray {
     UniformBlockArray& operator=(const UniformBlockArray&);
 
     /// Do something with each block
-    template <typename Func /* void(const StringIdentity, const UniformBlock&) */>
+    template <typename Func /* void(const size_t, const UniformBlock&) */>
     void visit(Func f) {
-        std::for_each(uniformBlockMap.begin(), uniformBlockMap.end(), [&](const auto& kv) {
-            if (kv.second) {
-                f(kv.first, *kv.second);
+        std::for_each(uniformBlockVector.begin(), uniformBlockVector.end(), [&](const auto& block) {
+            if (block) {
+                f(0, *block);
```

Comment:
Did you want to send the UBO index? If it's never used perhaps remove the first parameter entirely?

Reply:
Fixed.